pkg/importer/openapi.go

```diff
@@ -461,11 +492,20 @@ func (l *loader) findResponse(ref *openapi3.ResponseRef) *openapi3.Response {
 func (l *loader) buildParams(params openapi3.Parameters) Parameters {
 	out := Parameters{}
 	for _, param := range params {
+		var paramType Param
 		if param.Ref != "" {
-			out.Add(l.globalParams.items[strings.TrimPrefix(param.Ref, "#/components/parameters/")])
+			paramType = l.globalParams.items[strings.TrimPrefix(param.Ref, "#/components/parameters/")]
 		} else {
-			out.Add(l.buildParam(param.Value))
+			paramType = l.buildParam(param.Value)
 		}
+
+		// Cant have a sequence/set in the sysl params, so convert to a type alias
```

Comment:
I believe we can have a sequence/ set in sysl params e.g `SecondEndpoint(input <: sequence of Server.Request):`